### PR TITLE
JPEG 2000 message constants

### PR DESCRIPTION
**Pull request by marhop**
Concerning #140 GitHub issue. Most of the work should be done, but still some fine tuning needed.

### CODE REVIEW

**Comment by marhop** [comment]
That's all. If I haven't missed anything, all message strings are now centralized in the MessageConstants enum. Fixes #140

File: jhove-modules/src/main/java/edu/harvard/hul/ois/jhove/module/jpeg2000/ContCodestream.java [hunk 1]
```diff
@@ -98,7 +98,8 @@ public ContCodestream (Jpeg2000Module module,
     public boolean readCodestream (Codestream cs, RepInfo info) 
                 throws IOException
     {
-        final String badStream = "Ill-formed codestream";
+        final String badStream =
```
Comment (carlwilson) [inline jhove-modules/src/main/java/edu/harvard/hul/ois/jhove/module/jpeg2000/ContCodestream.java:101]: I think I'd leave the variable and just replace its use with the constant.

File: jhove-modules/src/main/java/edu/harvard/hul/ois/jhove/module/jpeg2000/MessageConstants.java [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    public static final String ERR_FRAGMENT_LIST_BOX_TABLE_SIZE_INVALID =
+        "Fragment Table has invalid length";
+    public static final String ERR_FRAGMENT_TABLE_BOX_CONTENT_INVALID =
+        "Invalid fragment table";
+    public static final String ERR_IMAGE_HEADER_BOX_COMPONENTS_EMPTY =
+        "ImageHeader Box has zero components";
+    public static final String ERR_IMAGE_HEADER_BOX_CONTEXT_INVALID =
+        "ImageHeader Box in illegal context";
+    public static final String ERR_IMAGE_HEADER_BOX_SIZE_INVALID =
+        "Image Header Box is incorrect size";
```
Comment (carlwilson) [inline jhove-modules/src/main/java/edu/harvard/hul/ois/jhove/module/jpeg2000/MessageConstants.java:102]: Consistency on Image Header vs. ImageHeader

**Review by carlwilson — APPROVED** [review]
I realise that you were consistent with the original messages. Going to approve this, thanks very much

**Pull request merged**